### DISCUSSION TITLE
Does Wiki.js has connectors / apps ?

### DISCUSSION THREAD

**Discussion by zak39** [requarks/wiki#2954] [Help / Questions] [answered]
Hello everyone :)

My Question : Does Wiki.js has connectors / apps ?

Details : Does it is possible to connect Wiki.js to GitLab or GitHub ? Nextcloud documents ? Just like Atlassian's Confluence app.

If it's not present. Do you envisage to put them in your roadmap ?

Thank you and sorry if this question already post ^^'

Best regards,

Zak,

**Comment by NGPixel** [comment, marked as answer]
See https://wiki.js.org/modules for the list of supported integrations.

Yes you can connect to GitLab and GitHub.

**Reply by zak39** [reply]
Thanks @NGPixel ! :)

I already saw the list of modules supported. But that concern the Authentication part only, no ?


> Let your users use their existing logins on your wiki, whether it's a social login or enterprise authentication.

I don't know, but you can join issues, commit, or others elements to a Wiki.js document ?
